drivers/mailbox/bcm2836/init.c

```diff
@@ -207,6 +207,20 @@ NTSTATUS RpiSetDeviceMacAddress (
         REG_SZ,
         &macAddrStr,
         sizeof(macAddrStr));
+    if (!NT_SUCCESS(status)) {
+        RPIQ_LOG_ERROR(
+            "Failed to set MAC value at NetworkAddress registry %!STATUS!",
+            status);
+        //goto End;
+    }
+	
+	    status = RtlWriteRegistryValue(
```

Comment:
So we write MAC address to both 0001 and 0003? Wouldn't this cause a MAC address conflict if both NICs end up attaching to the network?